packages/app-mobile/components/screens/search.js

```diff
@@ -123,7 +123,8 @@ class SearchScreenComponent extends BaseScreenComponent {
 
 		if (!this.isMounted_) return;
 
-		const parsedQuery = await SearchEngine.instance().parseQuery(query);
+		// ignore brackets when finding words to highlight
+		const parsedQuery = await SearchEngine.instance().parseQuery(query.replace(/[()]/g, ' '));
```

Comment:
This regex shows up 4 times in the code. Instead, could you create a specific function that returns what you need. I can't think of any good name for it, but maybe even "parseQueryNoBrackets" would be better. Then this of course would call `parseQuery` internally.